library/connect.c

```diff
@@ -615,12 +621,11 @@ static void ziti_disconnect_async(struct ziti_conn *conn) {
         case Connected:
         case CloseWrite:
         case Timedout: {
-            message *m = create_message(conn, ContentTypeStateClosed, 0);
             NEWP(wr, struct ziti_write_req_s);
             wr->conn = conn;
             wr->cb = on_disconnect;
```

Comment:
shouldn't `wr->close` be set here?

Reply:
thanks. I swear it was being set at one point.